mods/ts/sequences/infantry.yaml

```diff
@@ -140,6 +140,7 @@ oxanna:
 slav:
 	Inherits: ^BasicInfantry
 	Inherits@attack: ^BasicInfantryAttack
+	icon: xxicon
```

Comment:
#12751 and #12731 fix the problem, so can you please rebase on top of these (after they're merged) and the move the icon overrides back to the map?